src/index.ts

```diff
@@ -405,6 +406,14 @@ class Degit extends EventEmitter {
 	 * @param dest - The destination directory where the repository will be cloned.
 	 */
 	public async clone(dest: string) {
+		try {
+			execSync('git --version', { stdio: 'ignore' });
+		} catch (e) {
+			throw new DegitError(`could not find git. Make sure it is installed.`, {
```

Comment:
Maybe suggest verifying that `git`'s parent directory is available in the `PATH` env.

Reply:
I'm not sure I quite understand what you mean by git's parent directory. Different environments can have the git executable in different directories.

Surely if execSync try is a success git executable must be in one of the directories listed in the PATH env?

Please correct me if i misunderstood :)

Reply:
That is the point. `git` might be already installed but not properly added to `PATH`. You need to verify that the `PATH` includes the parent directory to `git` for it to be found by your OS.

Reply:
My comment refers only to the error message itself, which suggests checking if `git` is installed. If `git` is already is installed, the message wouldn't be as helpful because it doesn't address the actual problem in that `git` isn't available via `PATH`.

Reply:
Now I understand. Thank you Tony. I will change the message to be better.